pkg/core/storage/boltdb_store.go

```diff
@@ -108,8 +107,9 @@ func (s *BoltDBStore) Seek(key []byte, f func(k, v []byte)) {
 		}
 		return nil
 	})
+
 	if err != nil {
-		log.Error("error while executing seek in boltDB")
+		panic(err)
```

Comment:
hm, now it's panic.. I guess this one should be in separate commit since it changes logic

Reply:
+1.